daisy/resource_test.go

```diff
@@ -21,11 +21,11 @@ import (
 
 func TestExtendPartialURL(t *testing.T) {
```

Comment:
TestNormalize...

I would also split them

Reply:
Not modifying this anymore